src/action.js

```diff
@@ -204,24 +211,46 @@ async function main() {
   const { githubEventName, githubEventData } = await getEventData();
   core.endGroup();
 
+  return { config, githubEventName, githubEventData };
+}
+
+async function generate(data) {
+  const { config, githubEventName, githubEventData } = data;
+
   await core.group("Generate Map Data", async () => {
     const excludeLangs = [];
-    if (isForkedPullRequestEvent(githubEventName, githubEventData)) {
-      core.info("Detected Forked PR, disabling python");
+    if (
+      isForkedPullRequestEvent(githubEventName, githubEventData) &&
+      isInsecureConfiguration(config)
+    ) {
+      core.info(
+        "Detected Forked PR with potential insecure configuration, disabling python"
+      );
+      core.info(
+        "Consider updating your workflow to the latest version from CodeSee."
```

Comment:
Should we explain how to do this? Link to documentation?

Reply:
Filed https://codesee.atlassian.net/browse/CODESEE-1856 to track this work